src/main/java/dk/kb/present/util/DsPresentClient.java

```diff
@@ -216,7 +138,8 @@ public ContinuationInputStream<Long> getRecordsJSON(String origin, Long mTime, L
             throws IOException {
         URI uri;
         try {
-            uri = new URIBuilder(serviceURI + "records")
+            uri = new URIBuilder(serviceURI)
+                    .appendPath("records")                                        
                     // .setPath overrides all paths given in the serviceURI. Should be changed using pathSegments in the future
                     //.setPath("records")
```

Comment:
please remove this comment as it is not valid with the ned version of URIBuilder

Reply:
done